controllers/comments.controllers.js

```diff
@@ -24,3 +25,14 @@ exports.postCommentByArticleId = (req, res, next) => {
       next(err);
     });
 };
+
+exports.deleteCommentById = (req, res, next) => {
+  const { comment_id } = req.params;
+  removeCommentById(comment_id)
+    .then(() => {
+      res.status(204).send();
```

Comment:
This is absolutely fine to do, but a nicer alternative is using res.sendStatus(204) which doesnt include an empty send method.